dask/array/core.py

```diff
@@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
     return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
 
 
+def from_value(value, shape, dtype=None, name=None):
```

Comment:
OK, agreed on `name`, particularly if you're likely to actively use it in `xray`.

I still think that `value` is pretty general.  For example I can imagine people using `from_value` with numpy arrays or integers, claiming that they too are values.  Given that this function is probably fairly niche and more likely to be used within internal code I don't mind it being somewhat longer.  I would probably go for something like `from_dask_value` or `from_lazy_value`.